src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Register</title>
+    <link rel="stylesheet" href="/css/main.css">
+      <style>
+        label {
+          display: inline-block;
+          width: 100px;
+        }
+      </style>
+  </head>
+  <body>
+
+    <nav>
+      <a id="navTitle" href="/">CodeU Chat App</a>
```

Comment:
I recommend changing the title here too

To be consistent with "CodeU Team 14 Chat App"

Reply:
done!